Library/Formula/kitchen-sync.rb

```diff
@@ -0,0 +1,23 @@
+class KitchenSync < Formula
+  desc "Fast efficiently sync database without dumping & reloading"
+  homepage "https://github.com/willbryant/kitchen_sync"
+  url "https://github.com/willbryant/kitchen_sync/archive/0.37.tar.gz"
+  sha256 "a78206cdd3ac4d4aa646cdfc8ae34c5df87179223c59d5dcca625760c68ae4e7"
+  head "https://github.com/willbryant/kitchen_sync.git"
+
+  depends_on "cmake" => :build
+  depends_on "boost"
+  depends_on :mysql => :recommended
+  depends_on :postgresql => :optional
```

Comment:
It should also `depends_on "openssl"`, as mentioned in the [upstream docs](https://github.com/willbryant/kitchen_sync/blob/master/INSTALL.md#installing-kitchen-sync).